GooglePlayInstant/LoadingScreen/LoadingBarGenerator.cs

```diff
@@ -22,20 +22,23 @@ namespace GooglePlayInstant.LoadingScreen
     /// </summary>
     public static class LoadingBarGenerator
     {
-        // Loading bar size as a proportion of the screen size. Adjust if needed.
-        private const float LoadingBarWidthProportion = 0.7f;
-        private const float LoadingBarHeightProportion = 0.02f;
+        // Loading bar size.
+        private const float LoadingBarWidth = 672;
+        private const float LoadingBarHeight = 54;
 
         // Loading bar placement as a proportion of the screen size relative to the bottom left corner. Adjust if needed.
         private const float LoadingBarPositionX = 0.5f;
-        private const float LoadingBarPositionY = 0.3f;
+        private const float LoadingBarPositionY = 0.3984375f;

```

Comment:
This degree of precision feels a little unnecessary. Would 0.4 or 0.398 work as well?

Reply:
If we use lower precision we end up with a fractional value for position which makes our loading bar not pixel aligned on 1920x1080 devices. I can add a comment `// Corresponds to a position of 765 on 1080x1920 devices` or I can change the way this value is represented.

Reply:
Please add a comment along those lines.

Reply:
Resolved